src/Microsoft.AspNetCore.Mvc.Core/Formatters/InputFormatter.cs

```diff
@@ -20,32 +15,23 @@ namespace Microsoft.AspNetCore.Mvc.Formatters
     public abstract class InputFormatter : IInputFormatter, IApiRequestFormatMetadataProvider
     {
         /// <summary>
-        /// Returns UTF8 Encoding without BOM and throws on invalid bytes.
-        /// </summary>
-        protected static readonly Encoding UTF8EncodingWithoutBOM
-            = new UTF8Encoding(encoderShouldEmitUTF8Identifier: false, throwOnInvalidBytes: true);
-
-        /// <summary>
-        /// Returns UTF16 Encoding which uses littleEndian byte order with BOM and throws on invalid bytes.
-        /// </summary>
-        protected static readonly Encoding UTF16EncodingLittleEndian
-            = new UnicodeEncoding(bigEndian: false, byteOrderMark: true, throwOnInvalidBytes: true);
-
-        /// <summary>
-        /// Gets the mutable collection of character encodings supported by
-        /// this <see cref="InputFormatter"/>. The encodings are
-        /// used when reading the data.
-        /// </summary>
-        public IList<Encoding> SupportedEncodings { get; } = new List<Encoding>();
-
-        /// <summary>
         /// Gets the mutable collection of media type elements supported by
         /// this <see cref="InputFormatter"/>.
         /// </summary>
         public MediaTypeCollection SupportedMediaTypes { get; } = new MediaTypeCollection();
 
+        /// <summary>
+        /// Gets the default value for a given type.
+        /// </summary>
+        /// <param name="modelType">The type of the value.</param>
+        /// <returns>The default value for the <paramref name="modelType"/> type.</returns>
         protected object GetDefaultValueForType(Type modelType)
```

Comment:
Not related to your changes but should this be virtual? In what scenarios do we call this?